build.gradle

```diff
@@ -112,3 +131,32 @@ tasks.docker.dependsOn tasks.build
 task buildImage {
     dependsOn tasks.docker
 }
+
+sourceSets {
+    integrationTest {
+        java {
+            compileClasspath += main.output + test.output
+            runtimeClasspath += main.output + test.output
+            srcDir 'src/itest/java'
+        }
+        resources.srcDir 'src/itest/resources'
+    }
+}
+
+task itestSetup(type: Exec) {
```

Comment:
Nothing to do in `itestSetup`/`itestTeardown` ?

Reply:
No yet indeed :)

Reply:
That was on purpose in order to keep same structure than other projects but we can remove it yes, done :+1: